frontend/app/js/script.js

```diff
@@ -29,11 +29,18 @@ $(function() {
     $("a[data-action='enlarge']",codeDiv).show();
   });
 
-  $('form input[type=submit], form button[type=submit]').on('click', function() {
-    var $this = $(this);
+  $('form').on('submit', function() {
+    var $this = $(this).find('button');
```

Comment:
Ah! I see what you mean. I think I found a solution and will push it shortly. I want to click around some more to test it. Thanks!